1-js/06-advanced-functions/04-var/article.md

```diff
@@ -277,7 +279,7 @@ In all the above cases we declare a Function Expression and run it immediately.
 
 There are two main differences of `var` compared to `let/const`:
 
-1. `var` variables have no block scope, they are visible minimum at the function level.
+1. `var` variables have no block scope, they are visibility is scoped to current function, or global, if declared outside function.
```

Comment:
Please re-read your sentence: "they are visibility is scoped"?

Reply:
Fixed. It is "their visibility is scoped".